### DISCUSSION TITLE
If select "yes" then display different inputs?

### DISCUSSION THREAD

**Discussion by ecdeaver** [dabapps/django-forms-dynamic#14] [Q&A]
Pretty new to django and this project. 

Can this be used to create a dynamic form that for instance as the option of "is this associated with a date": "yes/no" if "yes", display a datetime input, if "no" don't display another input field? 

I understand the make/model example but not sure how to move beyond that.

**Comment by j4mie** [comment]
Hi, I've converted this to a discussion as it didn't really feel like an "issue" as such.

The short answer is yes: this library could be used to implement that behaviour, along with a suitable frontend implementation (either pure JavaScript or HTMX/Unpoly or similar).
